.eslintrc.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     "eqeqeq": [ "error", "always" ],
     "func-call-spacing": [ "error", "never" ],
     "indent": [ "error", 2 ],
-    "linebreak-style": [ "error", "unix" ],
+    "linebreak-style": 0,
```

Comment:
Would it be possible to change this back to error. I would hate for those nasty linebreaks to sneak into the codebase.

Reply:
This was an accident, wasn't meant to get checked in